api/controllers/admin.py

```diff
@@ -459,12 +463,14 @@ def get_pricing():
 
 
 @router.route('/social-media', methods=['GET'])
+@adminRequired
```

Comment:
This can't be set as admin because non-admin users also need to have access to the social media links.